hail/src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -154,13 +154,7 @@ object VEP {
                       fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
                   }
                 } else {
-                  val jv = try {
-                    JsonMethods.parse(s)
```

Comment:
As we discussed, this should probably stay, but downstream code should get fixed to produce a missing annotation when the JSON is invalid.

Then we should get @konradjk to set up docker image for a up to date version of VEP with Loftee which fixes the gene NaN bug.